dts/bindings/power/st,stm32-wkup-pin1.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2023 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  STM32 system wakeup Node with polarity configuration
+
+compatible: "st,stm32-wkup-pin1"
```

Comment:
`pin1` is confusing when the feature is exactly about selecting pin numbers.
`"st,stm32-wkup-pin-1cell"` would be clearer.
But in the end, why not be consistent with other drivers, when we use series  names to distinguish IP versions: `"st,stm32f0-wkup-pin"` ...